docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,302 @@
+anomaly forcing and (#2) ocean thermal forcing for projection runs, modern
+climatology files for (#3) atmospheric and (#4) ocean forcing, (#5) ISMIP6 basin
+number file, and (#6) the ocean basal-melt parameter files.
+
+Except for the file #3, The ISMIP6 source data (files #1, 2, 4-6) can be obtained by contacting the ISMIP6 steering
+committee as described `here. <https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica#A2.2_Retrieving_datasets_and_uploading_your_model_output>`_
+Once the users get access to the data on `Globus <https://www.globus.org>`_,
+and within the base path directory that the user specifies in the config file
+(i.e., the config option ``base_path_ismip6``;
+see :ref:`landice_ismip6_forcing_config`), they
+need to manually download the files following the directory structure
+and names provided in the GHub endpoints (named ``GHub-ISMIP6-Forcing``
+for the 2100 CE projection protocol and ``ISMIP6-Projections-Forcing-2300``
+for the 2300 CE projection protocol). That is, the directory paths and the file
+names must exactly match (even the letter case) those that are provided by the
+GHub endpoints. For example, if a user wants to process the atmosphere (SMB)
+forcing representing the ``SSP585`` scenario from the ``UKESM1-0-LL`` model
+provided by the ISMIP6-2100 protocol (i.e., from the ``GHub-ISMIP6-Forcing``
+Globus endpoint), they must create the directory path
+``AIS/Atmosphere_Forcing/UKESM1-0-LL/Regridded_8km/`` in their local system
+where they will download the file ``UKESM1-0-LL_anomaly_ssp585_1995-2100_8km.nc``.
+Note that the users only need to download SMB anomaly files (not the climatology
+file) that cover the period from 1995 to ``period_endyear``
+(either 2100 or 2300, defined in the config file;
+see :ref:`landice_ismip6_forcing_config`).
+
+Equivalently, for the
+ocean forcing, the user should create the directory path
+``AIS/Ocean_Forcing/ukesm1-0-ll_ssp585/1995-2100/`` and download the file
+``UKESM1-0-LL_ssp585_thermal_forcing_8km_x_60m.nc`` from the same endpoint.
+Users do not need to download the thermal
+forcing files for the years previous to 1995 as only the files downloaded from
+``1995-{period_endyear}`` will be processed.
+Also note to be ware that unlike those in the ``GHub-ISMIP6-Forcing`` endpoint,
+the directory names in the ``ISMIP6-Projections-Forcing-2300`` endpoint have a
+lower case "f" for the ``AIS/Atmospheric_forcing/`` and ``AIS/Ocean_forcing/``.
+
+In addition to atmospheric and ocean thermal forcing files that
+correspond to specific climate model (e.g., UKESM1-0-LL, CCSM4) and scenarios
+(e.g., SSP585, RCP85, RCP26-repeat), modern
+climatology files are needed. For the ``atmosphere`` testcase, users
+need to download the RACMO modern climatology data
+``RACMO2.3p2_ANT27_smb_yearly_1979_2018.nc`` from
+`here. <https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-albany-landice/>`_
```

Comment:
Since this is in the compass database, I don't think we want users to download this themselves.  Instead, compass should do int automatically and put it in its local, shared database.